db/postgres/scripts/create_db

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
```

Comment:
call it create_db.sh, to be more specific, or does k8s prefer this way?

Reply:
thanks, updated. 
-- k8s YAML file can detect either way, but yeah, it's more informative for a developer to do so